src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -7381,10 +7381,10 @@ def merge_asof(
         SnowflakeQueryCompiler
         """
         # TODO: SNOW-1634547: Implement remaining parameters by leveraging `merge` implementation
-        if left_index or right_index or tolerance or suffixes != ("_x", "_y"):
+        if tolerance or suffixes != ("_x", "_y"):
```

Comment:
So it was already supported? We're just removing the error messages in this PR?

Reply:
Yes we use a utility function as part of join, this PR just adds the tests and removes the error messages

Reply:
I see. Then can you please mention that in the description and point to the original PR where the actual support was added?

Reply:
Yep done